### ISSUE TITLE
UI: How to create boolean attributes using named arguments?

### ISSUE THREAD

**Issue by daattali** [rstudio/py-shiny#271]
To create a boolean HTML attribute, the dictionary method of attributes works. But I could not figure out how to create a boolean attribute using a named argument.

**Comment by cpsievert** [comment]
You can provide an empty string (e.g. `div(foo="")`)

**Comment by daattali** [comment]
True, that does create a valie HTML boolean attribute. I was hoping to have a way to create it in the form of `<div foo>` where the attribute name is completely dropped - I know functionally they're equivalent, so if there's no way to do this, you can close.
